rclcpp/test/benchmark/CMakeLists.txt

```diff
@@ -4,9 +4,12 @@ get_target_property(
   performance_test_fixture::performance_test_fixture
   INTERFACE_INCLUDE_DIRECTORIES)
 
+find_package(test_msgs REQUIRED)
```

Comment:
This is already included in the rclcpp/test/CMakeLists.txt. Is it customary to include it again?

Reply:
solved https://github.com/ros2/rclcpp/pull/1413/commits/2a6c761af3737dedce40293ba96c98e301b13aae